ethcore/Cargo.toml

```diff
@@ -14,7 +14,7 @@ build = "build.rs"
 ansi_term = "0.9"
 bit-set = "0.4"
 bloomchain = "0.1"
-bn = { git = "https://github.com/paritytech/bn" }
+bn = { git = "https://github.com/cdetrio/bn", branch = "mont-repr-fix" }
```

Comment:
could you make a PR to the upstream version so we still only have a paritytech dependency?